pkgs/applications/misc/fitnesstrax/default.nix

```diff
@@ -0,0 +1,51 @@
+{ pkgs
+, fetchFromGitHub
+, stdenv
+, rustPlatform
+, atk
+, gnome2
+, cairo
+, glib
+, gdk_pixbuf
+, gtkd
+, gtk3
+, wrapGAppsHook
+}:
+rustPlatform.buildRustPackage rec {
+  name = "fitnesstrax";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "luminescent-dreams";
+    repo = "fitnesstrax";
+    rev = "fitnesstrax-0.1.0";
+    sha256 = "1k6zhnbs0ggx7q0ig2abcnzprsgrychlpvsh6d36dw6mr8zpfkp7";
+  };
+
+  buildInputs = [
+    atk
+    gnome2.pango
```

Comment:
:( sucks that you have to use the old version of pango

Reply:
Yeah, doesn't appear that there's a new derivation. But since I don't use it directly, I don't so much notice.